guides/doc-Administering_Red_Hat_Satellite/topics/Disabling_Weak_Encryption.adoc

```diff
@@ -1,151 +1,41 @@
+You might want to change the encryption settings for {Project} depending on the security requirements of your infrastructure or to fix vulnerabilities quickly.
 
-.Disabling Weak SSL 2.0 and SSL 3.0 Encryption for {Project}
+Apache and Qpid services in {Project} have TLS 1.0 and 1.1 encryption enabled by default. Use this procedure on {Project} and {SmartProxy} to configure {Project} and {SmartProxy} to only allow TLS 1.2, as required, for example, by PCI-DSS regulations.
 
-To disable weak encryption for {Project}, complete the following steps:
+.Procedure
 
 . Open the `/etc/foreman-installer/custom-hiera.yaml` file for editing:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # vi /etc/foreman-installer/custom-hiera.yaml
 ----
-+
-. Add the following entries:
+
+. Add the following entry:
 +
 [options="nowrap"]
 ----
-# Foreman Proxy
-foreman_proxy::tls_disabled_versions: [ '1.1' ]
-
-# Dynflow
-foreman_proxy::plugin::dynflow::tls_disabled_versions: [ '1.1' ]
-
 # Apache
 apache::mod::ssl::ssl_protocol: [ 'ALL' , '-SSLv3' , '-TLSv1' , '-TLSv1.1' , '+TLSv1.2' ]
 
-# Tomcat / Candlepin
-candlepin::tls_versions: [ '1.2' ]
-
 # QPID Dispatch
-foreman_proxy_content::qpid_router_ssl_protocols: [ 'TLSv1.2' ]
 foreman_proxy_content::qpid_router_ssl_ciphers: 'ALL:!aNULL:+HIGH:-SSLv3:!IDEA-CBC-SHA'
 ----
-+
-. Rerun the `{foreman-installer}` tool with no arguments:
-+
-[options="nowrap" subs="+quotes,attributes"]
-----
-# {foreman-installer}
-----
-+
-. Restart the `{foreman-maintain}` services:
-+
-[options="nowrap" subs="+quotes,attributes"]
-----
-# {foreman-maintain} service restart
-----
-
-
-.Disabling Weak SSL 2.0 and SSL 3.0 Encryption for {SmartProxy}
 
-To disable weak encryption for {SmartProxy}, complete the following steps:
-
-. Open the `/etc/foreman-installer/custom-hiera.yaml` file for editing:
+. Enter the `{foreman-installer}` command to configure Qpid to use TLS 1.2 only:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# vi /etc/foreman-installer/custom-hiera.yaml
-----
-+
-. Add the following entries:
-+
-[options="nowrap"]
+# {foreman-installer} \
+--foreman-proxy-content-qpid-router-ssl-protocols=TLSv1.2
 ----
-# Foreman Proxy
-foreman_proxy::tls_disabled_versions: [ '1.1' ]
 
-# Dynflow
-foreman_proxy::plugin::dynflow::tls_disabled_versions: [ '1.1' ]
-
-# Apache
-apache::mod::ssl::ssl_protocol: [ 'ALL' , '-SSLv3' , '-TLSv1' , '-TLSv1.1' , '+TLSv1.2' ]
-
-# QPID Dispatch
-foreman_proxy_content::qpid_router_ssl_protocols: [ 'TLSv1.2' ]
-foreman_proxy_content::qpid_router_ssl_ciphers: 'ALL:!aNULL:+HIGH:-SSLv3:!IDEA-CBC-SHA'
-
-# PULP
-pulp::ssl_protocol: "ALL -SSLv3 -TLSv1 -TLSv1.1 +TLSv1.2"
-----
-+
-. Rerun the `{foreman-installer}` tool with no arguments:
-+
-[options="nowrap" subs="+quotes,attributes"]
-----
-# {foreman-installer} 
-----
-+
 . Restart the `{foreman-maintain}` services:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # {foreman-maintain} service restart
 ----
```

Comment:
The installer should restart any services. Not sure why you would restart them again after.
```suggestion
```